pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -339,6 +349,10 @@ func (m *manager) GetAllocatableCPUs() cpuset.CPUSet {
 	return m.allocatableCPUs.Clone()
 }
 
+func (m *manager) GetAllCPUs() cpuset.CPUSet {
+	return m.allCPUs.Clone()
+}
+
```

Comment:
Thinking aloud (so totally not blocking) we will need some refactoring cleanup about available/allocatable/all/reserved cpusets in the future